tasks/task_124_winowhy_wrong_reason_generation.json

```diff
@@ -0,0 +1,316 @@
+
+{
+    "Contributors": [
+        "Xinran Zhao",
+        "Hongming Zhang",
+        "Yangqiu Song"
+    ],
+    "Source": [
+        "winowhy"
+    ],
+    "Categories": [
+        "Incorrect Answer Generation"
+    ],
+    "Definition": "In this task you need to give wrong reasons to justify the pronoun coreference relations. The given sentences are provided by a sentence with a target pronoun and a sentence that justifies which noun phrase the pronoun refers to. The resolution of the pronoun coreference relations typically involve one or multiple following knowledge types about commonsense: First: 'Property', the knowledge about property of objects (e.g., ice is cold). Second: 'Object', the knowledge about objects (e.g., cats have ears). Third: 'Eventuality', the knowledge about eventuality (e.g., 'wake up' happens before 'open eyes'). Forth: 'Spatial', the knowledge about spatial position (e.g., object at the back can be blocked). Fifth: 'Quantity', the knowledge about numbers (e.g., 2 is smaller than 10). Sixth: all other knowledge if above ones are not suitable. Semantically relevant sentence could also be an wrong reason.",
+    "Positive Examples": [
+        {
+            "input": "Sentence: The city councilmen refused the demonstrators a permit because they feared violence. Question: Why does the 'they' refer to the city councilmen?",
```

Comment:
Looks like in the examples, there are `\n` before `Question`. 
Let's make sure that we follow the same pattern in the examples.

Reply:
And make sure that the examples don't appear among the instances (i.e., no information leakage from the examples to the instances).

Reply:
Thank you very much for your review. I followed the template of task 010 created by Swaroop, where there is a \n before every question. There is also a \n when we publish similar questions to human annotators so I thought it would be OK.

Reply:
I am neither encouraging nor discouraging the use of `\n`. 
My point is: 
 - you instances have `\n`
 - your examples do not have `\n` 
Make them consistent. 

Does that make sense?

Reply:
Yes, I will solve the issue now.